packages/firestore/src/exp/database.ts

```diff
@@ -460,6 +463,56 @@ export function terminate(firestore: FirebaseFirestore): Promise<void> {
   return firestore._delete();
 }
 
+/**
+ * Loads a Firestore bundle into the local cache.
+ *
+ * @param firestore - The `Firestore` instance to load bundles for for.
+ * @param bundleData
+ *   An object representing the bundle to be loaded. Valid objects are `ArrayBuffer`,
+ *   `ReadableStream<Uint8Array>` or `string`.
+ *
```

Comment:
This documentation style (newline versus dash) seems a bit inconsistent.

Reply:
Done.